CesiumGltfWriter/include/CesiumGltf/Writer.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+#include "CesiumGltf/Model.h"
+#include "WriteFlags.h"
+#include "WriteGLTFCallback.h"
+#include "WriterLibrary.h"
+#include <cstdint>
+#include <string_view>
+#include <vector>
+
+namespace CesiumGltf {
+
+/**
+ * @brief Write a glTF or glb asset to a byte vector.
+ *
+ * @param model Final assembled glTF asset, ready for serialization.
+ * @param flags Bitset flags used to control writing glTF serialization
+ * behavior.
+ *
+ * @details Serializes the provided model object into a byte vector using the
+ * provided flags to convert. Must provide WriteFlags::GLTF or WriteFlags::GLB.
+ * Cannot provide both simultaneously. There are a few special scenarios with
+ * serializing \link CesiumGltf::Buffer \endlink and \link CesiumGltf::Image
```

Comment:
Nitpicky, but the rest of our code use the jsdoc style: `{@link CesiumGltf::Buffer}` rather than `\link CesiumGltf::Buffer`.

Reply:
Changed.